joomla/joomla-cms
# Pass required parameter to canEditState.

Original report by user ernebenan on joomla-bugs.de: http://www.joomla-bugs.de/forum/index.php/topic,734.0.html

### Summary of Changes
The `JModelAdmin::canEditState` method requires one non-optional argument `$record`. As the parent method does not use the argument, it could be omitted, but PHP (at least starting from 7.0) throws a warning:

> Warning:  Missing argument 1 for JModelAdmin::canEditState(), called in /administrator/components/com_content/models/article.php on line 211 and defined in/libraries/legacy/model/admin.php on line 680

(PHP 7.0)

respectively an exception

> Too few arguments to function JModelAdmin::canEditState(), 0 passed in /administrator/components/com_content/models/article.php on line 211 and exactly 1 expected

(PHP 7.1)

### Testing Instructions
* Requirements: 
  * PHP 7.1 (to get the exception)
  * Backend template: Hathor (because Isis processes the ordering request via AJAX, so you won't see the error)
* Navigate to "Content - Featured Articles".
* Order by "Ordering".
* Try to change the ordering of the articles, e.g. moving one up or down.


### Expected result
* Re-ordering should work without any errors.


### Actual result
* On PHP 7.1, the action fails with the previously mentioned exception.

### Solution
* Apply this patch and test again.
